plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/glue/TestDeltaLakeRenameToWithGlueMetastore.java

```diff
@@ -43,7 +43,7 @@ protected QueryRunner createQueryRunner()
             throws Exception
     {
         Session deltaLakeSession = testSessionBuilder()
-                .setCatalog(CATALOG_NAME)
+                .setCatalog(DELTA_CATALOG)
```

Comment:
DELTA_CATALOG is the default, just omit the .setCatalog call

Reply:
It leads to the following failure:
```
Error:  Errors: 
Error:    TestDeltaLakeRenameToWithGlueMetastore>AbstractTestQueryFramework.init:113->createQueryRunner:53 » QueryFailed line 1:1: Catalog 'catalog' not found
```